core/src/com/google/inject/internal/InjectorImpl.java

```diff
@@ -101,7 +101,7 @@ public String toString() {
     NO_JIT,
     /** allows existing just in time bindings, but does not allow new ones */
     EXISTING_JIT,
-    /** allows existing just in time bindings & allows new ones to be created */
+    /** allows existing just in time bindings & allows new ones to be created in the current injector */
```

Comment:
This is wrong/misleading; as long as `requireExplicitBindings()` isn't set, `NEW_OR_EXISTING_JIT` still allows them in parent injectors.  Will fix.